### ISSUE TITLE
Behaviour of filtering on ID seems strange (to me :-))

### ISSUE THREAD

**Issue by thepilot21** [saltcorn/saltcorn#2718]
If I open a view on a table I can then filter that view by doing something like ?customer=fred. In which case the view is filtered as I would expect showing only rows where the customer field is "fred".

If I filter by ?id=5 I would expect to only see row with id of 5 displayed but instead the whole table remains displayed but row 5 is highlighted - is that really the way it should work - seems rather inconsistent IMHO. And if that is working as designed then how can I just filter to see one row based on ID.

Thanks,

Mike

**Comment by glutamate** [comment]
yes. It was done this way as a special case in order to support ListShowList (where you don't want the list to collapse to one item when filtering on ID, only to highlight it. You are right it is inconsistent. Will do something about this for next release
